examples/image_classification/image_classification_peft_lora.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+LOG_DIR="./logs"
+OUTPUT_DIR="./outputs"
+mkdir -p $LOG_DIR
+mkdir -p $OUTPUT_DIR
+
+model=${1:-"google/vit-base-patch16-224-in21k"}
```

Comment:
Is this vit-base model the only available option? If not, you should prepare a model list within this script or from separated txt file or something for other options.

Reply:
yes, I made a shell script to run all models.